src/Microsoft.DotNet.Cli.Utils/PathUtility.cs

```diff
@@ -119,6 +119,11 @@ public static string GetRelativePath(DirectoryInfo directory, FileSystemInfo chi
         /// </summary>
         public static string GetRelativePath(string path1, string path2)
         {
+            if (!Path.IsPathRooted(path1) || !Path.IsPathRooted(path2))
```

Comment:
I still cannot remove this. It is used by testframework and it is netstandard. But at least I can add some guard

Reply:
Another con I can think of is we cannot dogfood fix netstandard, netcore projects.